### PR TITLE
Add average bsq price

### PR DESCRIPTION
**Pull request by chimp1984**

### CODE REVIEW

**Comment by chimp1984** [comment]
Screen Shot 2019-09-06 at 04 37 27

File: desktop/src/main/java/bisq/desktop/util/GUIUtil.java
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/util/GUIUtil.java:1032]: We do have a mixture of in the codebase right now, but I would set BSQ always in uppercase to be able to differentiate it quicker from Bisq. Besides that utACK

Comment (chimp1984) [reply desktop/src/main/java/bisq/desktop/util/GUIUtil.java:1032]: You mean getBSQInUsd instead of getBsqInUsd? In the DAO where it appears often we dont use uppercase. In general we dont use all uppercase (getApiKey not getAPIKey) for  abbreviation as the following Camelcase letter might make the abbreviation harder to read. Of course question of tase and opinon, but code base is follwing that style largely.

Comment (ripcurlx) [reply desktop/src/main/java/bisq/desktop/util/GUIUtil.java:1032]: I just wanted to point it out as I found both variations in the codebase and we should stick to one or the other.

File: core/src/main/java/bisq/core/offer/OfferUtil.java
Comment (freimair) [inline core/src/main/java/bisq/core/offer/OfferUtil.java]: please only change stuff that actually changes something. Makes it more cumbersome to review...

File: core/src/main/java/bisq/core/offer/OfferUtil.java
Comment (freimair) [inline core/src/main/java/bisq/core/offer/OfferUtil.java]: please only change stuff that actually changes something. Makes it cumbersome to review...

File: core/src/main/java/bisq/core/offer/OfferUtil.java
Comment (freimair) [inline core/src/main/java/bisq/core/offer/OfferUtil.java]: please only change stuff that actually changes something. Makes it cumbersome to review...

File: desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java
Comment (freimair) [inline desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java]: please only change stuff that actually changes something. Makes it more cumbersome to review...

File: desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java
Comment (freimair) [inline desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java]: please only change stuff that actually changes something. Makes it more cumbersome to review...

File: desktop/src/main/java/bisq/desktop/util/GUIUtil.java
Comment (ripcurlx) [reply desktop/src/main/java/bisq/desktop/util/GUIUtil.java:1032]: @sqrrm @freimair as maintainers what's your opinion on that so we have a rule going forward?

File: desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java
Comment (freimair) [inline desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java]: please only change stuff that actually changes something. Makes it more cumbersome to review...

File: desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java
Comment (freimair) [inline desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java]: please only change stuff that actually changes something. Makes it more cumbersome to review...

File: desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java
Comment (freimair) [inline desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java:273]: please only change stuff that actually changes something. Makes it really cumbersome to review...

File: desktop/src/main/java/bisq/desktop/util/GUIUtil.java
Comment (freimair) [inline desktop/src/main/java/bisq/desktop/util/GUIUtil.java]: please only change stuff that actually changes something. I am stopping at this point because that is too much.

File: desktop/src/main/java/bisq/desktop/util/GUIUtil.java
Comment (freimair) [inline desktop/src/main/java/bisq/desktop/util/GUIUtil.java]: please only change stuff that actually changes something. Makes it REALLY cumbersome to review...

**Review by freimair — CHANGES_REQUESTED** [review]
Please only change stuff that actually changes code behavior. It is really cumbersome to check method signatures and whether they changed or not. Your suggestion contains 10+ method headers that have not changed functionally, but all of them are changes. Please revert those unnecessary changes now and in the future. Thanks!

File: desktop/src/main/java/bisq/desktop/util/GUIUtil.java
Comment (freimair) [reply desktop/src/main/java/bisq/desktop/util/GUIUtil.java:1032]: Java is CamelCase, most of our code-base is CamelCase, why do something different here. Looks ugly both ways :)

Comment (ripcurlx) [reply desktop/src/main/java/bisq/desktop/util/GUIUtil.java:1032]: Ok, than lets leave it in camelCase.

File: core/src/main/java/bisq/core/offer/OfferUtil.java
Comment (chimp1984) [reply core/src/main/java/bisq/core/offer/OfferUtil.java]: We all should apply the formatting rules at commit. In IntelliJ there is a checkbox for reorganize import and reformat. If all contributors use that we have same formatting in all committed classes. You can also compare directly in IntelliJ which is much more user friendly as a terminal diff IMO.

**Comment by chimp1984** [comment]
@freimair I try to clean up classes I touch. As siad above appying the formatting rules should be mandatory anyway to all commits IMO.

**Comment by freimair** [comment]
yes, applying formatting rules IS mandatory but only for new or functionally changed code. In the end, everything should be the same, however, it takes a lot more time reviewing changes that are no changes. So you might encounter longer times until stuff gets merged.

**Review by freimair — APPROVED** [review]
Ack

aand we shall not forget to get these changes into master...

**Comment by ripcurlx** [comment]
> Ack
> 
> aand we shall not forget to get these changes into master...

After the release I'll rebase and merge this branch into master anyways.

**Pull request merged**
